mobile/assets/i18n/en-US.json

```diff
@@ -566,5 +566,6 @@
   "version_announcement_overlay_title": "New Server Version Available \uD83C\uDF89",
   "viewer_remove_from_stack": "Remove from Stack",
   "viewer_stack_use_as_main_asset": "Use as Main Asset",
-  "viewer_unstack": "Un-Stack"
+  "viewer_unstack": "Un-Stack",
+  "get_my_preferences_error_message": "⚠️ Cannot get user preferences, please make sure the server version is up to date"
```

Comment:
We probably don't need this level of specificity in an error message. Something like "error_incompatible_server" would be fine with a message of "Please make sure the server is up to date".